token/cli/src/main.rs

```diff
@@ -1433,28 +1403,21 @@ async fn command_close(
     recipient: Pubkey,
     bulk_signers: BulkSigners,
 ) -> CommandResult {
-    let mint_pubkey = if !config.sign_only {
```

Comment:
this is a breaking change but i think its a merited one given how much nicer it makes the client function for close_account. tbh tho the more i interact with sign_only the more i want to restrict it to like, mint burn transfer approve revoke. something like that

Reply:
aaa sorry, i forgot to revert this when i reverted my (as it turned out misguided) `close_account` changes